### DISCUSSION TITLE
Did it recognize when a folder is moved to another path?

### DISCUSSION THREAD

**Discussion by fohnbit** [abraunegg/onedrive#1302] [Q&A] [answered]
Hello,

did the program recognize when I move one folder to another?
So he just need a move and not delete and add.

Because the folder has around 300GB and don't want to download again all.

thank you!

**Comment by abraunegg** [comment, marked as answer]
@fohnbit 
The application will & should detect correctly files that are added which are already present in the desired location.

Always test with `--dry-run` to confirm the application will perform the actions you are expecting.

**Reply by fohnbit** [reply]
thank you!
